tests/raft_scenarios/retire

```diff
@@ -0,0 +1,50 @@
+start_node,0
+assert_is_primary,0
+emit_signature,2
+
+assert_is_primary,0
+assert_commit_idx,0,2
+
+trust_node,2,1
+emit_signature,2
+
+dispatch_all
+periodic_all,10
+dispatch_all
+assert_commit_idx,0,4
+# Initial network state with nodes 0 and 1
+
+# Reconfig [0, 1] -> [0, 2]
+swap_node,2,0,2
+emit_signature,2
+
+# Leader replicates
+periodic_one,0,10
+dispatch_one,0
+
+# But node 1 runs for election before ack/cmt
+periodic_one,1,150
+dispatch_all
+dispatch_all
+assert_commit_idx,1,4
+
+# Tries to sign/extend, without success
+emit_signature,3
+dispatch_all
+dispatch_all
+assert_commit_idx,1,4
+
+emit_signature,3
+dispatch_all
+dispatch_all
+assert_commit_idx,1,4
+
+emit_signature,3
+dispatch_all
+dispatch_all
+assert_commit_idx,1,4
+
+emit_signature,3
+dispatch_all
+dispatch_all
+assert_commit_idx,1,4
```

Comment:
I think we need `periodic`s` in here so that the new primary actually emits AEs?

Reply:
Yes, sorry, fixed, but does not affect result.